UnityProject/Assets/Scripts/Weapons/Gun.cs

```diff
@@ -122,7 +123,7 @@ public class Gun : NetworkBehaviour, IPredictedCheckedInteractable<AimApply>, IC
 		/// <summary>
 		///     The amount of projectiles expended per shot
 		/// </summary>
-		public int ProjectilesFired;
+		public int ProjectilesFired = 1;
```

Comment:
if this is meant to be set in inspector, use a private property with ``SerializeField`` decorator and a public getter. If that's not the case, then hide this public from inspector with the ``HideInInspector`` decorator.